packages/fxa-auth-server/lib/senders/emails/layouts/subplat/index.scss

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../global.scss';
+
+.body {
+  max-width: 640px !important;
+  margin: global.$s-0 auto !important;
+}
+
+.header-container {
+  @extend .mb-6;
+}
+
+.fxa-logo {
+  padding: global.$s-5 global.$s-0 !important;
+  height: 56px !important;
+  width: 171px !important;
+  display: block !important;
+}
+
+.mozilla-logo {
+  padding: global.$s-8 global.$s-0 !important;
+  height: 34px !important;
+  width: 120px !important;
+  display: block !important;
+  margin: 0 auto !important;
+}
+
+.footer-container {
+  background-color: global.$black;
+  padding: global.$s-10 global.$s-6 !important;
+  @extend .mt-6;
+}
+
+.footer-text div {
+  @extend .text-sm;
+  @extend .font-sans;
+  color: global.$white !important;
+  text-align: center !important;
+}
+
+.footer-text-bottom-margin {
+  @extend .footer-text;
+
+  div {
+    @extend .mb-3;
+  }
+}
+
+.footer-link {
+  color: global.$white !important;
+}
```

Comment:
Do you think we should enforce consistency between the FxA and Subplat headings/body text? If we're going to match to parity here, I think FxA might need `text-body` and `text-header` scoped there and then different styles here since (especially the header text) they look fairly different. I'm also OK with poking a designer and asking them about making those things consistent now though.

Reply:
I'm also leaning towards parity for now, I've added a comment to the confirmation doc

Reply:
For parity, I think you need to move `text-body` and `text-header` into the FxA styles and then add the same classes with SubPlat styles to that subplat stylesheet. Still seeing FxA styles here:

<img width="824" alt="image" src="https://user-images.githubusercontent.com/13018240/134196808-5dc1c484-8837-4f52-a5c6-0201640b6ea8.png">

Reply:
Actually/hopefully, I think the only thing you need to move to FxA styles is `@extend .font-sans` and the `text-align center` for those two classes and then set those in subplat styles. Maybe we can get away with sharing the font-size but if not we'll have to move that as well

Reply:
It ended up being a little more substantial just because the classes beneath `%text-body-common` are already constructed by the time the styles in the fxa-specific stylesheet are constructed, so overriding `%text-body-common` further down wouldn't have effect. Open to feedback for sure!

Reply:
I'll see if there's ways to improve this in future PRs but I don't want to block template conversions so seems fine to me